provisioning/cnit-correlator/deployment.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -xeuo pipefail
+IFS=$'\n\t'
+
+prelude_profile="prelude-correlator"
+prelude_registrator_passwd="password"
+prelude_manager_host="10.10.1.44"
```

Comment:
This should be a parameter

Reply:
Hello @ferrarimarco 
Into the scissor variable in Vagrantfile, should the cnit-correlator VM follow a specific order, can i put it simply after KAFKA_PRELUDE_CONNECTOR_VM_NAME ?

Reply:
Vagrant starts the VMs in the order you list them in the Vagrantfile. What are the dependencies of your VM in terms of other components? In other words: should some components be already available when your component starts? If so put your VM after the needed ones.